src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -561,7 +572,7 @@ public enum DiamondPearlPromos implements LogicCardInfo {
             onAttack {
               damage 60
 
-              if (confirm(text)) {
+              if (self.cards.energyCount(L) && self.cards.energyCount(M) && confirm(text)) {
```

Comment:
See, this is still valid for a single Rainbow Energy, but that is not correct.

Reply:
I've changed it to use `self.cards.energySufficient(L, M)` instead